### PR TITLE
Added privlevel to User Details

### PR DESCRIPTION
**Pull request by poixen**
Fix for #2342

Before:
screen shot 2017-01-06 at 20 33 18

After:
screen shot 2017-01-06 at 22 16 24

### CODE REVIEW

**Comment by Nightfirecat** [comment]
Would personally prefer to see `<userLevel> (<privLevel>)`, like **Registered user (DONATOR)**; at minimum, there should be spaces around the vertical bar, not just after it imo.

**Comment by poixen** [comment]
Thanks for feedback @Nightfirecat!
@woogerboy21 @Daenyth and myself had a talk about it on the gitter page and came up with this version.

**Comment by tritoch** [comment]
I think `Donor` is the proper English word versus `Donator`

**Comment by woogerboy21** [comment]
http://www.thefreedictionary.com/donator

```
donator - noun
A person who gives to a charity or cause:
```

**Comment by poixen** [comment]
@woogerboy21 I pull the string from the `ServerUser_Info`. So it can be whatever you like, dont you just update the DB?

**Comment by woogerboy21** [comment]
> I pull the string from the ServerUser_Info. So it can be whatever you like, dont you just update the DB?

Yes.

File: cockatrice/src/userinfobox.cpp [hunk 1]
```diff
@@ -110,6 +110,11 @@ void UserInfoBox::updateInfo(const ServerInfo_User &user)
         userLevelText = tr("Registered user");
     else
         userLevelText = tr("Unregistered user");
+
+    if (user.has_privlevel() && user.privlevel() != "NONE") {
+        userLevelText += tr(" | %1").arg(user.privlevel().c_str());
```
Comment (tooomm) [inline cockatrice/src/userinfobox.cpp]: Can you exclude " | " from tr() please? Actually there is nothing to translate at all here... ;P

**Comment by tooomm** [comment]
Do you save them as all capital letter in the db @woogerboy21? --> DONATOR and VIP?
Just wondering. :)

Cool pr @poixen! Pretty busy today, he? 👍

**Comment by woogerboy21** [comment]
> Do you save them as all capital letter in the db?

Yes they are stored in all caps in the DB.

**Pull request merged**